### ISSUE TITLE
Start using "v" prefix for tags

### ISSUE THREAD

**Issue by glyn** [deislabs/duffle#865]
This is for consistency with cnab-go.

Also https://semver.org/spec/v1.0.0.html#tagging-specification-semvertag favours this:
> When tagging releases in a version control system, the tag for a version MUST be “vX.Y.Z” e.g. “v3.1.0”.

**Comment by thaney071** [comment]
Your developer guide page points to the semversion 2.0.0 webpage, but your issue is for a 1.0.0 part of the spec. 2.0.0 has removed this suggestion and now has this in the FAQ:
https://semver.org/#is-v123-a-semantic-version

If you are intending to use semver 1.0.0, then the link might need to be updated. Otherwise, it would be better to state this is for consistency with other parts of the cnab project.

**Comment by carolynvs** [comment]
@thaney071 Correct, as Glyn said we would like to be consistent with the other cnabio projects which use the v prefix for our git tags.

We do follow semver 2 for the CNAB spec for bundle version by the way, the bundle.json version field doesn't use a v prefix. This issue is only applicable to the git tag for this repo.
